src/views/LoginView.vue

```diff
@@ -137,44 +137,34 @@
         </v-row>
       </v-container>
     </section>
-    
-    <!--///////////////////////// 
+
+    <!--/////////////////////////
      ///BlockQuote Section////
     /////////////////////////-->
     <section>
       <v-container fluid class="quote text-h5 text-center d-flex flex-column align-center">
-        <blockquote>
-          "Cuttle is a sharp, fast game built entirely on excellent mechanics. It is the sort of game - had I
-          known about it in college - I would have worn decks ragged through play"
-        </blockquote> 
-        <br> 
-        <cite>Richard Garfield - <br>Creator of Magic: The Gathering</cite>
```

Comment:
I did remove this br here. If we need a line break, we need to make a new localized slug, or an interpolated slug so we can dynamically add the break tag. Barring that, I just removed it since it's more of a CSS thing.